examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,216 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scroll_view/custom_scroll_view.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Widget tree is visible', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    expect(
+      find.byType(Scaffold),
+      findsOne,
+      reason: 'Expected to have a Scaffold in the App',
+    );
```

Comment:
```suggestion
    expect(find.byType(Scaffold), findsOne);
```

Adding 'reason' is not bad in itself but I think we should reserved it to non-obvious cases because:
- it makes the code harder to read by splitting it on several lines.
- it took more effort to write (and might lead to typos).
- it takes more effort to read and then realized that this sentence just restate the code.
- it takes more effort to review (looking for typos, formatting issues, understanding if the comment is right).

This is similar to over-commenting: it starts from a good intention but leads to readibility and maintenance issues.
Reason texts and comments are welcome when they add value.